src/ipfs.rs

```diff
@@ -81,7 +81,7 @@ impl ContentAddressedStorage for Ipfs {
         // TODO find a way to stream this better? (use .take with max block size?)
         let block: Block = Block::encode(RawCodec, Code::Blake3_256, content)?;
         let cid = self.put_block(block).await?;
-        self.insert_pin(&cid, false).await?;
+        self.insert_pin(&cid, true).await?;
```

Comment:
The blocks stored by this API are all `raw` encoded and therefore don't have children, so this doesn't really change anything?